### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by lex111**
В основном, это автоправки, но теперь уже в блоках кода (pre-элементы), но есть подправленные места в обычном тексте.

### CODE REVIEW

**Comment by chrisdavidmills** [comment]
So this looks like it is failing because there are `javascript:` expressions found in /ru/docs/conflicting/Web/API/Web_Storage_API

Am I reading this right @peterbe ?

**Comment by lex111** [comment]
I ran the build command locally (from PR check) and its output shows that there are 44 `unsafe_html` flaws. I assume 2 flaws in changed file, means I have to fix 22 files (approximately) for the PR to be able to merge at the moment. I think it is unreasonable for such mass-edits PR.

image

File: files/ru/conflicting/learn/javascript/objects/index.html [hunk 1]
```diff
@@ -273,7 +273,7 @@ <h4 id="Inheritance" name="Inheritance">Наследование</h4>
 };
 
 // Создаём объект Student.prototype, который наследуется от Person.prototype.
-// Примечание: Рспространённая ошибка здесь, это использование "new Person()", чтобы создать
+// Примечание: Распространенная ошибка здесь, это использование "new Person()", чтобы создать
```
[the remark this answers is not in the document]
Comment (lex111) [reply files/ru/conflicting/learn/javascript/objects/index.html:276]: Используем, потом в отдельном PR ёфикацию проведу массово, сейчас в спеллчекере не стал включать эту опцию.

[the remark this answers is not in the document]
Comment (lex111) [reply files/ru/conflicting/learn/javascript/objects/index.html:276]: Вот только что слили небольшое руководство https://github.com/mdn/translated-content/blob/main/docs/ru/translation-guide.md

**Comment by peterbe** [comment]
This PR is actually safe. CC @chrisdavidmills 

See https://github.com/mdn/translated-content/pull/188#issuecomment-801352460

The "Unsafe HTML" errors don't actually come from the HTML itself. It's from some old pages that use the old `{{CompatibilityTable}}` macro. I'm fixing that in Yari in https://github.com/mdn/yari/pull/3270

**Comment by peterbe** [comment]
It would be nice to wait a little bit for Yari to upgrade that KS macro and the rebase *this* to be sure it works.

**Comment by peterbe** [comment]
@lex111 @chrisdavidmills Now that Yari has upgrade that old legacy KS macro, the tests are no longer failing here. Hurray!
(I clicked into the failing CI and pressed the "Re-run jobs" button)

**Comment by lex111** [comment]
@peterbe awesome, thank you for quickly fixing this!

**Pull request merged**